src/utils.js

```diff
@@ -17,6 +18,15 @@ module.exports = {
 // Utils
 /**************************/
 
+function firstDefined() {
+	for(var i = 0; i < arguments.length; i++) {
+		if(typeof arguments[i] !== 'undefined') {
+			return arguments[i];
+		}
+	}
+	return (function(x) { return x; })();
```

Comment:
Yeah you're right. I am coming from old-school JS where you couldn't rely on the undefined variable actually containing undefined.
(i.e. you used to be able to write `window.undefined = 1`.
I'll fix it up.